tests/dummy/app/templates/components/changeset-form.hbs

```diff
@@ -1,10 +1,14 @@
 <h2>Changeset Name: <span data-test-changeset-user-name>{{this.changeset.user.name}}</span></h2>
 <h2>Changeset Email: <span data-test-changeset-user-email>{{this.changeset.user.email}}</span></h2>
 <h2>Changeset cid: <span data-test-changeset-cid>{{this.changeset.cid}}</span></h2>
+<h2>Changeset notifications email: <span data-test-changeset-notifications-email>{{this.changeset.notifications.email}}</span></h2>
+<h2>Changeset notifications sms: <span data-test-changeset-notifications-sms>{{this.changeset.notifications.sms}}</span></h2>
 <br />
 <h2>Model Email: <span data-test-model-user-name>{{this.model.user.name}}</span></h2>
 <h2>Model Email: <span data-test-model-user-email>{{this.model.user.email}}</span></h2>
 <h2>Model cid: <span data-test-model-cid>{{this.model.cid}}</span></h2>
+<h2>Model notifications email: <span data-test-model-notifications-email>{{this.model.notifications.email}}</span></h2>
+<h2>Model notifications sms: <span data-test-model-notifications-sms>{{this.model.notifications.sms}}</span></h2>
```

Comment:
Can we confirm that using `changeset-get` helper in `master` makes the tests pass?

Reply:
Confirmed. Rebased with master and changed to use `changeset-get`. Bug still there.